.github/workflows/generate-artifacts.yml

```diff
@@ -0,0 +1,69 @@
+name: allcontributors-zenodo
+
+on:
+  pull_request: []
+
+jobs:
+  Update:
+    name: Generate
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+      - name: Setup test environment
+        run: conda create --quiet --name test requests
+
+      - name: Run Testing
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          export PATH="/usr/share/miniconda/bin:$PATH"
+          source activate test
+          /bin/bash tests/test_client.sh
+
+      - name: Generate Updated Zenodo and Contributors
+      
+        # Important! Update to release https://github.com/con/allcontributors-zenodo
+        uses: ./
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:        
+
+          # ".zenodo.json to update. If does not exist, must define zenodo_doi"
+          zenodo_file: .zenodo.json
+
+          # Zenodo DOI needed for init. Leave unset to skip init.
+          zenodo_doi: 10.5281/zenodo.1012531
```

Comment:
Given how hard it seems to be to generate an initial zenodo.json (I even found you on several issues!) I added this functionality because it's useful.

Reply:
yeap, as I stated in the summary - a one time to run command to initiate one from the existing info on zenodo would be useful, but I do not see it being used upon every PR etc (besides again -- for testing purposes). Or am I missing some use case?